### ISSUE TITLE
After setting navigateToLoginRequestUrl: true still redirecting to home page instead of requested url

### ISSUE THREAD

**Issue by asvashrith** [AzureAD/microsoft-authentication-library-for-js#5600]
### Core Library

MSAL.js v1 (@azure/msal or msal)

### Core Library Version

1.1.2

### Wrapper Library

MSAL Angular (@azure/msal-angular)

### Wrapper Library Version

none

### Public or Confidential Client?

Public

### Description

When we access our webpage with any direct URL(eg: locahost/home/admin) it'll redirect to microsoft login authentication(login redirect) after successful login it is redirecting to homepage (localhost/home). instead it should redirect to calling URL

### MSAL Configuration

```javascript
auth: {
            clientId: 'clientID',
            authority: 'authority',
            redirectUri: window.location.origin,
            protectedResourceMap: 'protectedResourceMap',
            navigateToLoginRequestUrl: true
        },
        cache: {
            //cacheLocation:'cacheLocation'
            cacheLocation:("localStorage" as CacheLocation), 
            storeAuthStateInCookie:isIE 
        }
```

### Identity Provider

Azure AD / MSA

### Source

External (Customer)

**Comment by asvashrith** [comment]
https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/929 **facing similar issue**.

**Comment by jo-arroyo** [comment]
@asvashrith Please upgrade to msal-angular v2. There have been many updates to how redirects are handled.

**Comment by asvashrith** [comment]
Upgrading version needs lot of changes. Is there a way to fix this in existing "msal": "^1.4.4" and "@azure/msal-angular": "^1.1.2", @jo-arroyo @sameerag

**Comment by sameerag** [comment]
@asvashrith Unfortunately we only support the latest Angular version as there are too many backward compat issues in the framework. We are only supporting new work and enhancements in `msal@2.x` and other wrappers based on the same.
